server_src/lib/serverCommands.js

```diff
@@ -25,6 +30,45 @@ class SystemCommands {
     }
     return checkForNamedService;
   }
+  // This will need changing when .deb / installation script becomes a thing. It's built to deal with the current implementation.
+  static async updateOctoFarm() {
+    logger.info("Update OctoFarm server requests");
+    let haveWeSuccessfullyUpdatedOctoFarm = false;
+    let statusTypeForUser = null;
+    let messageForUser = null;
+    try {
+      // Check to see if current dir contains a git folder... hard fail otherwise.
+      let doWeHaveAGitFolder = await isGitSync("./");
+      if (!doWeHaveAGitFolder) {
+        let notGitRepoError = "Not a git repository, manual update required...";
+        messageForUser = notGitRepoError;
+        statusTypeForUser = "error";
+        throw notGitRepoError;
+      }
+      logger.info("Root location contain a git folder: ", doWeHaveAGitFolder);
```

Comment:
Log really necessary? Just a reminder that logging in general can be taxing on CPU / file IO.

Reply:
Not really as the rest of the function is more important I'll remove this in the next PR.

Reply:
I'll keep that in mind for the server.js split as well as there is an ungodly amount of stuff logged. I propose we just log errors or anything relavant to connections in files then and completely reduce the IO but I'll add that to the server.js projects bots as it's not relevant here.